### DISCUSSION TITLE
Make `AppIcon` a rounded-rectangle to match Big Sur style guide

### DISCUSSION THREAD

**Discussion by rakleed** [sparkle-project/Sparkle#2595] [Ideas]
The current [`AppIcon`](https://github.com/sparkle-project/Sparkle/tree/2.x/Resources/Images.xcassets/AppIcon.appiconset) doesn't match modern macOS style guides, so it doesn't look good against the background of other icons.

![CleanShot 2024-04-22 at 00 36 49](https://github.com/AdguardTeam/AdGuardVPNForMac/assets/19418601/131e8e47-d2a5-4572-9179-32438a48e332)

Could you please update the `AppIcon` for macOS according to Big Sur specifications?

More: https://developer.apple.com/design/human-interface-guidelines/app-icons#macOS

Production Templates: https://developer.apple.com/design/resources/#macos-apps

**Comment by zorgiepoo** [comment]
This AppIcon is only used for the test app which is for debugging Sparkle, not end users (at least since Sparkle 2). More importantly, the icon is used on the website. But I don't suspect it's going to be a high priority for people to volunteer updating the icon asset. It sure isn't going to get updated from a maintainer.
